x-pack/plugin/profiling/src/internalClusterTest/java/org/elasticsearch/xpack/profiling/GetFlameGraphActionIT.java

```diff
@@ -9,13 +9,13 @@
 
 public class GetFlameGraphActionIT extends ProfilingTestCase {
     public void testGetStackTracesUnfiltered() throws Exception {
-        GetStackTracesRequest request = new GetStackTracesRequest(10, null);
+        GetStackTracesRequest request = new GetStackTracesRequest(10, null, null, null);
         GetFlamegraphResponse response = client().execute(GetFlamegraphAction.INSTANCE, request).get();
         // only spot-check top level properties - detailed tests are done in unit tests
-        assertEquals(231, response.getSize());
+        assertEquals(297, response.getSize());
```

Comment:
The number changes because there is random downsampling and randomness depends on the hashcode of the request (which changes due to additional fields).